spk-config.yaml

```diff
@@ -6,13 +6,33 @@ azure_cli:
 
 # Azure DevOps Configuration
 azure_devops: # Should this live under the `spk` configuration?
-  access_token: "access-token" # This is your Personal Access Token with permission to modify and access this private repo. Ex. "${env:AZDO_PERSONAL_ACCESS_TOKEN}". Leave this empty if project is public
+  org: "epicstuff"
+  server_url:
+  orgUrl: "https://dev.azure.com/epicstuff"
+  project: "hellobedrock"
   hld_repository: "my-hld-repository"
   manifest_repository: "my-manifest-repository"
-  org: "my-az-do-org"
-  project: "my-mono-repo-project"
-  server_url:
-
+  access_token: "${env:AZDO_PERSONAL_ACCESS_TOKEN}" #Leave this empty if project is public
+  variable_group:
+    name: "myvg"
+    description: "myvg description"
+    variables: # Either this or Key Value provider should be configured for this variable group
+      var1:
+        value: value1
+        isSecret: true
+      var2:
+        value: value2
+        isSecret: false
+    key_vault_provider:
+      name: "sarathpvault" # name of the Azure Key Vaukt with Secrets
+      secrets: ["secret1", "secret2"] # Names of secrets from Azure Key Vault to map in Variable Group
+      service_connection: # service connection is required to authorize with Azure Key Vault
+        name: "Bedrock-GitOps-KeyVault" # if the service connection with this name does not exist, the following values are required to create a new service connection with this name
+        subscription_id: "${env:AZDO_SERVICE_CONNECTION_SUBSCRIPTION_ID}" # Azure Subscription id where Key Vault exist
+        subscription_name: "CSE Dev Crews Americas West" # Azure Subscription name where Key Vault exist
+        service_principal_id: "${env:AZDO_SERVICE_CONNECTION_CLIENT_ID}" # Service Principal Id that has 'Get' and 'List' in Key Vault Access Policy
+        service_principal_secret: "${env:AZDO_SERVICE_CONNECTION_CLIENT_SECRET}" # Service Principal secret for the above Service Principal Id
+        tenant_id: "${env:AZDO_SERVICE_CONNECTION_TENANT_ID}" # AAD Tenant Id for the above Service Principal
```

Comment:
Is there a need to have all of this be part of our sample `spk-config.yaml`? I know there was a discussion on minimizing the content of `spk-config.yaml`, and if there is no specific reason to have it be stored here, then we should remove it. An alternative would be to document some of this stuff in a README.

Reply:
same comment as @samiyaakhtar, haha.

Reply:
@yradsmikham, @samiyaakhtar - any ideas how to get this information in the runtime without passing these as arguments from a command? Too many arguments to pass from the command.

Reply:
These would be needed only during the onboard command, right?

Reply:
The actual command is not yet finalized but won't be very frequent similar to the onbaord command.